app/views/directives/workflow-arrow-buttons.js

```diff
@@ -630,7 +630,11 @@ const toasterMessages = mergeTranslationKeys(toasterMessagesTranslations);
                         $scope.updateStep(tab);
                     
                     if(tab == "intro"){
-                        loadArticle();
+                        $scope.$watch('documentType', function(newVal) {
```

Comment:
`documentType` should not chnaged one laoded.
Can we make it one-time watch? 
you simply prefix the name with `::`

```suggestion
                        $scope.$watch('::documentType', function(newVal) {
```